shell/layouts/default.vue

```diff
@@ -318,6 +318,14 @@ export default {
     overflow-y: auto;
     min-height: 0px;
 
+    &:has(.side-menu) {
+    padding-left: $app-bar-collapsed-width;
```

Comment:
comment only. in theory having this in the default layout and not other layouts should break those other layouts. that doesn't seem to be the case though, still investigating

Reply:
ok, i think default is really root and other layouts sit inside. apart from blank, unauthed and standalone. blank and unauthed don't contain the app bar. standalone ui's will use the normal layouts. the standalone layout is new, and not associated with the standalone uis